Week3/hw/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>HYF JS1 Week3 Classwork</title>
+  </head>
+  <body>
+    <header
+      style="font-size: 350%; color: lightseagreen; border: 5px dashed blue; text-align: center"
+    >
+      Hi, this page is for Js practice purpose!
+    </header>
+    <script src="app1.js"></script>
+    <script src="app2.js"></script>
```

Comment:
I think you also forgot `assignment 2`. See [here](https://github.com/HackYourFuture/JavaScript1/blob/master/Week3/MAKEME.md)